### ISSUE TITLE
nvda does not notice the return of focus to other areas of power point when clicking the formatting pinsel button

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14801]
### Steps to reproduce:
create a powerpoint presentation
walk to a field on the slide, type text, and enter formatting.
Now press alt, tab walk to Format Painter and press space.
Now press esc twice to put the focus back on the presentation.
### Actual behavior:
nvda announces unknown, and from this moment on, it does not read any more or a change of focus within the powerpoint, either when walking with tab or going to another slide with page down.
If you open the nvda menu and close it, or remove the focus from powerpoint and return with alt + tab, everything works again.
### Expected behavior:
nvda should announce focus changes within nvda when walking with the keyboard.
I don't have a newer office version to test, I don't know if this is due to the powerpoint version, or if it's something introduced in nvda recently, when creating a presentation in another newer powerpoint version a few months ago I didn't notice this behavior .
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-28043,14bc6bd2

#### Windows version:
10 19045.2728
#### Name and version of other software in use when reproducing the issue:
POWERPNT.EXE, Microsoft Office 2016 16.0.4266.1001

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[ppt.txt](https://github.com/nvaccess/nvda/files/11171522/ppt.txt)

**Comment by seanbudd** [comment]
Please report this to Microsoft

**Comment by douggeoffray** [comment]
I'm using PowerPoint version 2410 build 16.0.18029.20004 on Windows 23H2 build 22631.4037. For me, when I select the format Painter toggle button, focus moves back to the text area I was in. I entered some text, bolded a word, moved my cursor to the bold word and pressed Alt followed by several tabs to the button and pressed space. At that point I heard focus move back to the text area and I could arrow around and select other text.

However, I'm not seeing a way to set the formatting on the selected range as this approach seems to be mouse centric. Meaning if I use the mouse to select text and let go, the selected text immediately gets the formatting. This does seem like a bug from the keyboard unless I'm missing something.

However, you can use ctrl+shift+c to copy the current formatting into the clipboard, move or select the area you want the formatting applicated and press ctrl+shift+v. This works but doesn't give any confirmation of the copy or the paste.

I will file a bug for both issues. Can't use the keyboard with the button and you get no confirmation when using the ctrl+shift+c / ctrl+shift+v commands. But I would like to verify if you are still having the focus issue when you enable the button itself.

**Comment by fernando-jose-silva** [comment]
On a machine with the updated Office I don't have this problem. Although I think that in older versions of Office the problem may still exist, such as in Office 2016 with only security updates.

**Comment by seanbudd** [comment]
Closing as fixed in Office
